src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -840,18 +834,42 @@ private static bool IsInDocumentationComment(SyntaxNode node)
             return false;
         }
 
-        // Check each parameter for a default value containing span, and create an InitializerSemanticModel for binding the default value if so.
-        // Otherwise, return null.
-        private MemberSemanticModel GetOrAddModelForParameterDefaultValue(SeparatedSyntaxList<ParameterSyntax> parameterList, TextSpan span)
+        // Check parameter for a default value containing span, and create an InitializerSemanticModel for binding the default value if so.
+        // Otherwise, return model for enclosing context.
```

Comment:
Enclosing context here is the context for the code that contains the parameter declaration. The comment doesn't talk about binders and doesn't specify what should they be. From the behavior point of view. If we are inside parameter's default, we should create special model, otherwise return what we used to return before this change.